qiskit/circuit/library/generalized_gates/uc.py

```diff
@@ -34,13 +34,9 @@
 from qiskit.circuit.exceptions import CircuitError
 from qiskit.exceptions import QiskitError
 
-# pylint: disable=cyclic-import
-from qiskit.synthesis.one_qubit.one_qubit_decompose import OneQubitEulerDecomposer
-
 from .diagonal import Diagonal
 
 _EPS = 1e-10  # global variable used to chop very small numbers to zero
-_DECOMPOSER1Q = OneQubitEulerDecomposer("U3")
```

Comment:
This was never used, I removed it to make the cyclic import handling easier but if it is necessary, I can bring it back.